os_faults/common/service.py

```diff
@@ -84,7 +85,7 @@ def kill(self, nodes=None):
         nodes = nodes if nodes is not None else self.get_nodes()
         LOG.info("Kill '%s' service on nodes: %s", self.SERVICE_NAME,
                  nodes.get_ips())
-        cmd = {'kill': {'grep': self.GREP, 'sig': signal.SIGKILL}}
+        cmd = {'kill': {'grep': self.SERVICE_NAME, 'sig': signal.SIGKILL}}
```

Comment:
Do you think this will break other drivers?  Regardless it seems to be a completely required change...